lib/random.d.ts

```diff
@@ -37,25 +37,22 @@ declare module "@offirmo/random" {
 		static die(sideCount: number): (engine: Engine) => number;
 		static dice(sideCount: number, dieCount: number): (engine: Engine) => number;
 		static uuid4(engine: Engine): string;
-		static string(engine: Engine, length: number): string;
-		static string(pool: string, length: number): (engine: Engine, length: number) => string;
```

Comment:
Again, `Random.string` looks like `(pool?) => (engine, length) => string`. There is a default pool if one isn't provided.